File: cpp/Baju.cpp
#include "Aksesoris.cpp"

class Baju : public Aksesoris {
    public:
        Baju(int id, string nama, double harga, int stok, 
            string jenis, string bahan, string warna, 
            string untuk, string size, string merk)
            : Aksesoris(id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk) {}
    
        string getUntuk() const override { return PetShop::getUntuk(); }
        string getSize() const override { return PetShop::getSize(); }
        string getMerk() const override { return PetShop::getMerk(); }
    };

File: java/Baju.java
public class Baju extends Aksesoris {
    public Baju(int id, String nama, double harga, int stok,
              String jenis, String bahan, String warna,
              String untuk, String size, String merk) {
        super(id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk);
    }
    
    @Override
    public String getUntuk() { return super.getUntuk(); }
    
    @Override
    public String getSize() { return super.getSize(); }
    
    @Override
    public String getMerk() { return super.getMerk(); }
}